src/main/kotlin/no/nav/syfo/clients/KafkaConsumers.kt

```diff
@@ -13,9 +13,10 @@ fun kafkaConsumerSmregProperties(
 ) = Properties().apply {
     this[ConsumerConfig.GROUP_ID_CONFIG] = "${env.applicationName}-consumer4"
     this[ConsumerConfig.AUTO_OFFSET_RESET_CONFIG] = "earliest"
-    this[ConsumerConfig.MAX_POLL_RECORDS_CONFIG] = "50"
+    this[ConsumerConfig.MAX_POLL_RECORDS_CONFIG] = "10000"
+    this[ConsumerConfig.MAX_PARTITION_FETCH_BYTES_CONFIG] = "" + (10 * 1024 * 1024)
     this[CommonClientConfigs.RETRIES_CONFIG] = "2"
-    this["acks"] = "all"
```

Comment:
Hva gjør `acks`, og hvorfor trenger vi det ikke?

Reply:
Så vidt jeg har klart å finne ut er dette noe som er i bruk (lengre) - kan undersøke nærmere. Jeg tror det kommer noen warnings om at dette er en konfig Kafka ikke vet noe om.

Reply:
Se feks her: https://www.cloudkarafka.com/blog/what-does-in-sync-in-apache-kafka-really-mean.html

Så dette er en konfig som skal brukes for **producers**, ikke for consumers.

Reply:
Det samme gjelder konfig'en "retries", så jeg har fjernet den også.